segway/_util.py

```diff
@@ -189,11 +189,15 @@ def get_col_index(chromosome, trackname):
 
 
 def get_label_color(label):
-    color = SCHEME[label % NUM_COLORS]
+    color = SCHEME[hash_label(label) % NUM_COLORS]
 
     return ",".join(map(str, color))
 
 
+def hash_label(label):
+    return sum(ord(character) for character in label)
```

Comment:
Relatively minor but the hashlib functions are *probably* faster than this and it would simplify this code.

Reply:
In lieu of other things, to do, I tested this and the custom function is faster (though it will be called so few times that it doesn't matter). 
More importantly, the best option among hashlib functions would look like `int(hashlib.shale_256(label.encode()).hexdigest(10), 16)`. I personally don't think that's much simpler, though I will add a comment to explain the hash_label function.